ui/admin/app/templates/onboarding/success.hbs

```diff
@@ -0,0 +1,90 @@
+{{page-title (t 'titles.success')}}
+<Rose::Layout::Centered class='create-resources'>
+  <Rose::Frame>
+    <:header>
+      <h2>{{t 'onboarding.success.titles.connect'}}</h2>
+    </:header>
+    <:body>
+      {{! We have created }}
+      <div class='create-resources-item'>
+        <span class='create-resources-item-header'>
+          <Rose::Icon @name='flight-icons/svg/check-circle-16' />
+          <h3>{{t 'onboarding.success.titles.creation-success'}}</h3>
+        </span>
+        <p>{{t 'onboarding.success.descriptions.role'}}</p>
+      </div>
+
+      {{! Donwload Desktop client }}
+      <div class='create-resources-item'>
+        <span class='create-resources-item-header'>
+          <Rose::Icon @name='logo-app-background' @size='extra-large' />
+          <h3>{{t 'onboarding.success.titles.desktop'}}</h3>
+        </span>
+        <p>{{t 'onboarding.success.descriptions.download'}}</p>
+
+        <Hds::Link::Standalone
+          @icon='download'
+          @text='{{t "onboarding.success.links.install"}}'
+          @href='{{doc-url "downloads"}}'
+          isHrefExternal={{true}}
+        />
+
+        <Hds::Link::Standalone
+          @icon='docs-link'
```

Comment:
I don't think the quotes are necessary with text, href

Reply:
Thanks for the catch!! 
I realize I messed up this file was not consistent, I was mixing single and double quotes. Now unnecessary quotes are deleted and just using single quotes.